src/components/api/apiSlice.js

```diff
@@ -0,0 +1,73 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export const apiSlice = createApi({
```

Comment:
- In the end is your choice but, why add all stores, funcs, etc on the same script? According to best practices, it would be great if this is separate on diff scripts regarding the functionality, this makes the code better to understand and change in the future. But as I said in the beginning, is your choice 😄

Reply:
Hello [Victor](@vichuge), thank you for the review. There are a few things. We have the store in a different folder call `app` and the function also in the different repository called api. Kindly verify. Thank you

Reply:
@principlebrothers Thanks for stating this I want to further explain to @vichuge, that :
**Redux maintainers** now encourage the use of **Redux toolkit** over Redux,
and with that being said they recommend the  **DUCK Pattern** with RTQ Query 
to keep one base URL and all fetching from different endpoints in one file  `api.js`
Please confirm my words at: [Redux Official Docs ](https://redux.js.org/tutorials/essentials/part-2-app-structure)